src/screencast_keys/__init__.py

```diff
@@ -115,7 +116,7 @@ def call_silently(fn, *args):
 
 def register_screencast_keys_enable_property():
     def get_func(self):
-        return ops.SK_OT_ScreencastKeys.is_running()
+        return ops.is_running()
```

Comment:
As I mentioned the comment, is it possible to move to the classmethod of `SK_OT_ScreencastKeys`?

Reply:
Ok. moved it back to operator. Seems like it's working.